### DISCUSSION TITLE
Dynamic timestamp values in URL

### DISCUSSION THREAD

**Discussion by danlor** [grafana/grafana-infinity-datasource#84] [General]
Hi there

Many thanks for that great datasource. I'm quite new to grafana and I found that plugin which is really great.

What I try to achieve is to provide with the query an URL, that changes constantly, depending on the daytime. 
As I want to get back only what changed in the last 7 days for example from my API call I need to provide the time like

/threats?updatedAt__gt=2021-05-30T00:00:00

Is this possible and if yes can you please give me a hint how to achieve this?

kind regards
Dani

**Comment by yesoreyeram** [comment]
You need to use grafana's global variables described in https://grafana.com/docs/grafana/latest/variables/variable-types/global-variables/ 

Refer https://github.com/yesoreyeram/grafana-infinity-datasource/discussions/38 's "Using Grafana's time filter" section and comments there for example usage.

**Reply by danlor** [reply]
Many thanks for that information, really appreciated. I will give it a try.
